assets/aml-benchmark/scripts/custom_dataset_preprocessors/base_preprocessor_template.py

```diff
@@ -20,15 +20,12 @@ def _parse_args():
     return argss
 
 
-def _read_jsonl_file(file_path: str) -> List[Dict[str, Any]]:
-    """Read `.jsonl` file and return a list of dictionaries.
+def _read_input_file(file_path: str) -> List[Dict[str, Any]]:
+    """Read files that have content in the json format and return a list of dictionaries.
```

Comment:
This is confusing, we should keep the older function name. we can update the description to:

```
Read `.jsonl` file and return a list of dictionaries.

If the file does not have any extension, try to read it as a `.jsonl` file.
```